File: IT23173118-Lab06/fileimport.py
import csv

def print_header(csv_file):
    """Print the header of the CSV file"""
    with open(csv_file, 'r', newline='') as f:
        reader = csv.reader(f)
        header = next(reader, None)
        if header:
            print(','.join(header))

def print_contents(csv_file):
    """Print the contents of the CSV file (excluding header)"""
    with open(csv_file, 'r', newline='') as f:
        reader = csv.reader(f)
        next(reader, None)  # Skip header
        for row in reader:
            print(','.join(row))

# Example usage
if __name__ == "__main__":
    csv_file = "iris (1).csv"
    print("Header:")
    print_header(csv_file)
    print("\nContents:")
    print_contents(csv_file)

File: IT23173118-Lab06/fileimport.rb
require 'csv'

def print_header(csv_file)
  # Print the header of the CSV file
  header = CSV.read(csv_file, headers: false).first
  puts header.join(',')
end

def print_contents(csv_file)
  # Print the contents of the CSV file (excluding header)
  CSV.foreach(csv_file, headers: true) do |row|
    puts row.to_a.join(',')
  end
end


if __FILE__ == $PROGRAM_NAME
  csv_file = "iris (1).csv"
  puts "Header:"
  print_header(csv_file)
  puts "\nContents:"
  print_contents(csv_file)
end